tests/rptest/services/flink.py

```diff
@@ -202,36 +201,64 @@ def __init__(self, context, *args, **kwargs):
             self.STATE_FAILED, self.STATE_FINISHED, self.STATE_CANCELED
         ]
 
-        # Provider instance/node util class
-        nodeutils = ProviderInstanceUtils(self.logger)
-        # Get specs for an flink node
-        specs = nodeutils.get_node_specs(self.nodes[0], context)
-        self.logger.info(f"Using specs of '{specs}'")
-
-        # Handle service configuration for single node
-        # 10% ram - reserved
-        # 10% ram - job manager
-        # 80% ram / vcpus - per taskmanager
-        # Example, xlarge, 32 ram, 4 vcpus
-        # 32 * 0.1 = 3 (rounded down)
-        # 32 * 0.8 / vcpus = 6 (rounded down)
-        jm_ram = int(specs["ram"] * 0.1)
-        tm_ram = int(specs["ram"] * 0.8 // specs["vcpus"])
+        self.logger.info(f"Using specs: {num_cpus} cpus, {node_ram_mb}MB RAM")
         self.service_config = flink_config
-        # default, 1600m
-        self.service_config['jobmanager.memory.process.size'] = f"{jm_ram}g"
-        # default, 1728m
-        self.service_config['taskmanager.memory.process.size'] = f"{tm_ram}g"
-        # 1280m, total - 1g for JVM metadata, etc
-        self.service_config['taskmanager.memory.flink.size'] = f"{tm_ram-1}g"
+
+        tm_count = num_cpus
+        if node_ram_mb < 10240:
```

Comment:
Is the idea here to continue to get NO memory information from docker (0), and just assume low memory environment?  Please add a comment as I don't think it's entirely expected that `self.redpanda.get_node_memory_mb()` returns garbage sometimes (e.g. for docker).

Reply:
It will return limits and we should respect that as long as we do not want out container to be commanded to shrink first and then OOMed. 
Having a 10G value here is just a borderline where we can safely use previous formula (which we do not want to touch) as it is proven to be working solution for checks on 70M transactions in 5 min.